src/utils/git.ts

```diff
@@ -11,9 +11,9 @@ export async function getCurrentCommitInfo(): Promise<{
   const commitSha = await shell("git show -s --format=%H");
   const timestampStr = await shell("git show -s --format=%ct");
   const branchStr = await shell("git branch --show-current");
-  const timestamp = parseInt(timestampStr, 10);
+  const timestamp = Number.parseInt(timestampStr, 10);
```

Comment:
Same as for Infinity, we should remove this rule to keep it consistent with the other codebases